src/orion/client/runner.py

```diff
@@ -151,7 +195,8 @@ def __init__(
         max_trials_per_worker,
         max_broken,
         trial_arg,
-        on_error,
+        on_error=None,
+        prepare_trial=None,
```

Comment:
Same comment about the `prepare_trial` argument here.

Reply:
Fixed in 5c150f31